### PR TITLE
aff switch

### PR DESCRIPTION
**Pull request by flohump**
## :bird: Description of this PR :bird:

Added options for afforestation assumptions
Default settings remain unchanged
The new switches are especially useful in combination with dynamic forestry (off by default)

- **config** changed order of output scripts. Some functions in rds_report require gridded outputs.
- **32_forestry** added switch `s32_aff_prot`
- **56_ghg_policy** added two scenarios for GHG emission pricing and options for afforestation

## :wrench: Checklist for PR creator :wrench:

- [x] Label pull request [from the label list](https://github.com/magpiemodel/magpie/labels).
  - **Low risk**: Simple bugfixes (missing files, updated documentation, typos) or changes  in start or output scripts
  - **Medium risk**: Uncritical changes in the model core (e.g. moderate modifications in non-default realizations)
  - **High risk**: Critical changes in model core or default settings (e.g. changing a model default or adjusting a core mechanic in the model)

- [x] Self-review own code
  - No hard coded numbers and cluster/country/region names.
  - The new code doesn't contain declared but unused parameters or variables.
  - [`magpie4`](https://github.com/pik-piam/magpie4) R library has been updated accordingly and backwards compatible where necessary.
  - `scenario_config.csv` has been updated accordingly (important if `default.cfg` has been updated)

- [x] Document changes 
  - Add changes to `CHANGELOG.md`
  - Where relevant, put In-code documentation comments
  - Properly address updates in interfaces in the module documentations
  - run [`goxygen::goxygen()`](https://github.com/pik-piam/goxygen) and verify the modified code is properly documented

- [x] Perform test runs
  - **Low risk**: 
    - Run a compilation check via `Rscript start.R --> "compilation check"`
  - **Medium risk**: 
    - Run test runs via `Rscript start.R --> "test runs"`
    - Check logs for errors/warnings
  - **High risk**:
    - Run test runs via `Rscript start.R --> "test runs"`
    - Check logs for errors/warnings
    - Default run from the PR target branch for comparison
    - Provide relevant comparison plots (land-use, emissions, food prices, land-use intensity,...)

### :chart_with_downwards_trend: Performance changes :chart_with_upwards_trend:
  
  - This PR's default :  0.38h

## :rotating_light: Checklist for reviewer :rotating_light:

- PR is labeled correctly
- Code changes look reasonable
  - No hard coded numbers and cluster/country/region names.
  - No unnecessary increase in module interfaces
  - model behavior/performance is satisfactory.
- Changes are properly documented
  - `CHANGELOG` is updated correctly
  - Updates in interfaces have been properly addressed in the module documentations
  - In-code documentation looks appropriate
- [ ] content review done (at least 1)
- [x] RSE review done (at least 1)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +28,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - **config** changed default for `s_use_gdx` from 2 to 0
 - **scripts** added restart points after each time step from which the model can now be restarted if the simulation aborts at some point
 - **scripts** added SLURM dayMax submission type for standby QOS
+- **config** added options for afforestation assumptions and updated additional data to 4.38
+- **32_forestry** added switch `s32_aff_prot`
```
Comment (abhimishr) [inline CHANGELOG.md:32]: Perhaps a short description of what this switch does (similar to the one in main code)

**Review by abhimishr — CHANGES_REQUESTED** [review]
Seems fine to me :) except for skipping one revision in additional data but we have done that in the past as well (i think) and one minor comment about the changelog

**Review by pfuehrlich-pik — APPROVED** [review]

**Comment by flohump** [comment]
> Seems fine to me :) except for skipping one revision in additional data but we have done that in the past as well (i think) and one minor comment about the changelog

Version 4.37 already exists but has not yet been added to the develop.

File: CHANGELOG.md (hunk 1 above)
Comment (flohump) [reply CHANGELOG.md]: done.

**Review by abhimishr — APPROVED** [review]

**Pull request merged**